pkgs/applications/science/logic/surelog/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+, pkg-config
+, libuuid
+, openjdk11
+, gperftools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "surelog";
+  version = "v1.36";
```

Comment:
I learned in a review in one of my own packages by some nix maintainer that the preferred way in nix packages is to not include the 'v' here in the version name, but then assemble the full tag name below in the `rev` of the github repo.

```nix
version = "1.37"
# ... later
src = ... {
   ref = "v${version}";
   # ...
};
```